src/components/Kv/KvAccordionItem.vue

```diff
@@ -1,30 +1,26 @@
 <template>
 	<div
-		class="kv-accordion"
-		:class="{
-			'kv-accordion--open' : isOpen,
-		}"
+		class="tw-border-b tw-border-tertiary tw-relative"
 	>
 		<button
-			class="kv-accordion__header-button"
+			class="tw-w-full tw-flex tw-justify-between tw-items-center tw-py-[0.75rem] tw-px-0
+				tw-text-left disabled:tw-cursor-not-allowed disabled:tw-opacity-low"
 			@click.prevent="toggle"
 			:disabled="disabled"
 			:aria-controls="`kv-accordion-${id}`"
 			:aria-expanded="isOpen ? 'true' : 'false'"
 		>
-			<span class="kv-accordion__header-text">
+			<span class="tw-flex-1">
 				<slot name="header"></slot>
 			</span>
-			<kv-icon
-				class="kv-accordion__header-icon"
-				name="small-chevron"
-				:from-sprite="true"
-				aria-hidden="true"
+			<kv-material-icon
+				class="tw-h-3 tw-w-[25px] tw-transition tw-transform tw-duration-500 tw-ease"
```

Comment:
Same as above for `tw-w-[25px]`. Should either become `tw-w-3` or move to `style="width: 25px;`

Reply:
I'd prefer we move towards our size/spacing scale rather than custom values.

Reply:
+1 to using `tw-w-3`

Reply:
Good, value replaced.